### ISSUE TITLE
CMS 7 compatibility

### ISSUE THREAD

**Issue by pi-phi** [mneuhaus/TYPO3-Kickstarter#3]
please make kickstarter compatible to TYPO3 CMS 7

**Comment by pgampe** [comment]
What is currently broken? Did you try to use it with the compatibility6 extension?

**Comment by pi-phi** [comment]
kickstarter can be an intro for beginners to have a first running extension. 
as ext:compatibility6 will slow down your website it would be nicer to have an up to date extension-stub, which show the best way to beginn with extension programming.

**Comment by pgampe** [comment]
Well, that would require almost a rewrite of the complete code base. I think it would be better to invest this time into the extension builder.

**Comment by pi-phi** [comment]
which still is a big overhead if you just want to insert one new field to an existing table (as an example)
only 4 files needed (including ext_emconf.php), but a beginner will break it without a 'wizard' like kickstarter.

**Comment by mneuhaus** [comment]
i agree that it is quite an overhead and that it would be cool to have this with a updated codebase. 
but to be honest, this is currently not realistic to do, because, as pgampe said, it would require
a complete rewrite of the extension. but i dislike the extension builder approach quite a bit, maybe
i'll someday do it, but don't hold me to that.
